scripts/game_controller.js

```diff
@@ -0,0 +1,38 @@
+/* global  Player */
+const gameController = (function gameController(doc) { // eslint-disable-line no-unused-vars
+  const render = (arr) => {
+    const squares = [...doc.querySelectorAll('.square')];
+    squares.forEach((square, index) => {
+      square.textContent = arr[index];
+    });
+  };
+
+  const players = [];
+
+  const getPlayers = (playerOne, playerTwo) => [Player(
+    playerOne,
+    '●',
+  ),
+  Player(
+    playerTwo,
+    '✘',
+  )];
+
+  const currentPlayer = 0;
+
+  const displayOnTitleMessage = function displayOnTitleMessage(message) {
+    doc.getElementById('cPlayer').textContent = message;
+  };
```

Comment:
Don't forget we ought to separate DOM manipulation from application logic! 🙂 

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from pathwright.

Reply:
Hi Maya,
 First of all, thanks for the review.

The [gameController](https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/game/scripts/game_controller.js) module doesn't contain the application logic. This module contains methods to manipulate the DOM: `render` and `displayOnTitleMessage `
https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/b196e9001d92ec8a295a897c1cf6328b63b639fa/scripts/game_controller.js#L3-L8

https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/b196e9001d92ec8a295a897c1cf6328b63b639fa/scripts/game_controller.js#L23-L25

It also serves to store the `players`,  `currentPlayer` and if the user is playing `againstComputer `:

https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/b196e9001d92ec8a295a897c1cf6328b63b639fa/scripts/game_controller.js#L10-L19

https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/b196e9001d92ec8a295a897c1cf6328b63b639fa/scripts/game_controller.js#L21

https://github.com/JuanPabloGil/Tic-Tac-Toe-JS/blob/b196e9001d92ec8a295a897c1cf6328b63b639fa/scripts/game_controller.js#L27

Reply:
Hey, Sorry about this! The game controller ought to have application logic or a similar Game object. I must have misunderstood your intentions

Reply:
But this is fine, You may submit another review and you should get the 👍

Reply:
I've requested for a new code review. Thanks.